doc/source/configuration/ironic.rst

```diff
@@ -0,0 +1,230 @@
+    port_security
+    conntrack_helper
+    dns_domain_ports
+
+The neutron l3 agent also requires conntrack to be set as an extension in
+``kolla/config/neutron/l3_agent.ini``
+
+.. code-block:: ini
+
+  [agent]
+  extensions = conntrack_helper
+
+It is also required to load the conntrack kernel module ``nf_nat_tftp``, 
+``nf_conntrack`` and ``nf_conntrack_tftp`` on network nodes. You can load these
+modules using modprobe or define these in /etc/module-load.
+
+The Ironic neutron router will also need to be configured to use 
+conntrack_helper.
+
+.. code-block:: json
+
+  "conntrack_helpers": {
+    {"protocol": "udp",
+     "port": 69, 
+     "helper": "tftp"}
+  }
+
+Currently it's not possible to add this helper via the OpenStack CLI, to add
+this to the Ironic router you will need to make a request to the API directly,
+for example via cURL.
+
+.. code-block:: console
+
+  curl -g -i -X POST \
+  http://<internal_api_vip>:9696/v2.0/routers/<ironic_router_uuid>/conntrack_helpers \
+  -H "Accept: application/json" \ 
+  -H "User-Agent: openstacksdk/2.0.0 keystoneauth1/5.4.0 python-requests/2.31.0 CPython/3.9.18" \
+  -H "X-Auth-Token: <issued_token>" \ 
+  -d '{ "conntrack_helper": {"helper": "tftp", "protocol": "udp", "port": 69 } }'
+
+TFTP server
+===========
+
+By default the Ironic TFTP server (ironic_pxe container) will call the UEFI
+boot file ``ipxe-x86_64.efi`` instead of ``ipxe.efi`` meaning no boot file will
+be sent during the PXE boot process in the default configuration.
+
+As of now this is solved by using a hack workaround by changing the boot file
+in the ``ironic_pxe`` container. To do this you will need to enter the 
+container and rename the file manually.
+
+.. code-block:: console
+
+  docker exec ironic_pxe “mv /tftpboot/ipxe-x86_64.efi /tftpboot/ipxe.efi”
+
+Baremetal inventory
+===================
+
+To begin enrolling nodes you will need to define them in the hosts file.
+
+[r1]
+hv1 ipmi_address=10.1.28.16
+hv2 ipmi_address=10.1.28.17
+…
+
+[r1:vars]
+ironic_driver=redfish
+resource_class=<your_resource_class>
+redfish_system_id=<your_redfish_systen_id>
+redfish_verify_ca=<your_redfish_verify_ca>
+redfish_username=<your_redfish_username>
+redfish_password=<your_redfish_password>
+
+[baremetal-compute:children]
+r1
+
+The typical layout for baremetal nodes are separated by racks, for instance
+in rack 1 we have the following configuration set up where the BMC addresses
+are defined for all nodes, and Redfish information such as username, passwords
+and the system ID are defined for the rack as a whole.
+
+You can add more racks to the deployment by replicating the rack 1 example and
+adding that as an entry to the baremetal-compute group.
+
+Node enrollment
+===============
+
+When nodes are defined in the inventory you can begin enrolling them by
+invoking the Kayobe commmand (Note that only the Redfish driver is supported
+by this command)
+
+.. code-block:: console
+
+  kayobe baremetal compute register
```

Comment:
Not available yet pending this upstream [patch](https://review.opendev.org/c/openstack/kayobe/+/909671)